UI/window-basic-main.cpp

```diff
@@ -6931,9 +6996,6 @@ void OBSBasic::YoutubeStreamCheck(const std::string &key)
 		auto item = json["items"][0];
 		auto status = item["status"]["streamStatus"].string_value();
 		if (status == "active") {
-			QMetaObject::invokeMethod(ui->broadcastButton,
-						  "setEnabled",
-						  Q_ARG(bool, true));
```

Comment:
TODO: Found a situation when `ui->broadcastButton` is disabled while running the check.

Reply:
Done